### PR TITLE
Exercise 2: Fibonacci

### PR DESCRIPTION
**Pull request by kawapa**

### CODE REVIEW

File: module1/homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
-    // TODO: Your implementation goes here
-    return 0;
+    int first = 0;
+    int second = 1;
+    int temp;
+
+    if (sequence == 0)
+        return first;
+    if (sequence < 2)
+        return second;
```
Comment (Marcinati) [inline module1/homework/fibonacci/fibonacci.hpp:11]: You can return sequence if sequence is less than 2.

File: module1/homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
-}
+    if (sequence == 0)
+        return 0;
+    if (sequence < 3)
+        return 1;
```
Comment (Marcinati) [inline module1/homework/fibonacci/fibonacci.hpp:27]: Same here, you can return sequence is sequence is less than 2.

File: module1/homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -1,11 +1,31 @@
...
+
+    return fibonacci_recursive(sequence - 2) +
+        fibonacci_recursive(sequence - 1);
```
Comment (Marcinati) [inline module1/homework/fibonacci/fibonacci.hpp:30]: This will work also for sequence equals to 2.
fibonacci_recursive(0) + fibonacci_recursive(1) = 1

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1430], "points": [2, 6] }

**Pull request closed without merging**